andela/ah-sealteam-frontend
# #161278886 View bookmarked articles

#### What does this PR do?
An authenticated user should be able to see all his bookmarked articles in his/her profile and navigate to them
#### Description of Task to be completed?
When you navigate to profile and click bookmark article you will be able to see the artilces that you have book marked
#### How should this be manually tested?
Pull this project and navigate to this branch ft-bookmark-articles-161278886.
Make sure you have bookmarked articles, in this case with postman since it has not been implemented yet
#### Any background context you want to provide?
You can read about pagination of bookmarks here [react-pagination](https://www.npmjs.com/package/react-js-pagination)
#### What are the relevant pivotal tracker stories?
[#161278886](https://www.pivotaltracker.com/story/show/161278886)

#### Checklist:

- [ ] At least 4 people have reviewed my PR
